scripts/audit_frontend_licenses.py

```diff
@@ -129,6 +131,15 @@
         "http://colorbrewer2.org/",
         "Cynthia Brewer",
     ),
+    (
+        #Our own @streamlit/lib
+        '@streamlit/app',
```

Comment:
Comment says @streamlit/lib but entry is @streamlit/app

Reply:
Also, will we be able to remove this entry if we add a proper license to the app (or lib) directory? Can you add an inline TODO comment saying "# TODO: before merging to develop, add a LICENSE file and remove this entry"

Reply:
Hm let me just add a license to `@streamlit/lib` and see if this gets resolved.